tests/beta/interfaces/test_qnode_tf.py

```diff
@@ -52,12 +52,11 @@ def circuit(a):
 
         # without the interface, the tape simply returns an array of results
         assert isinstance(res, np.ndarray)
-        assert res.shape == (1,)
-        assert isinstance(res[0], float)
+        assert res.shape == tuple()
 
         # without the interface, the tape is unable to deduce
         # trainable parameters
-        assert circuit.qtape.trainable_params == {0, 1}
+        assert circuit.qtape.trainable_params == {0}

```

Comment:
Is this because now every parameter in the tape is assumed to be non-trainable, unless otherwise stated (see above)?